docs/upgrading_to_v8.0.md

```diff
@@ -4,7 +4,7 @@ The v8.0 release of *kubernetes-engine* is a backwards incompatible
 release.
 
 ## Workload Identity (beta)
-Beta clusters now have Workload Identity enabled by default. To disable Workload Identity, set `identity_namespace = null`
+Beta clusters now have Workload Identity enabled by default. To disable Workload Identity, set `workload_pool = null`
```

Comment:
Was this incorrect for v8.0? We shouldn't change old docs to match the current version.

Reply:
done